### ISSUE TITLE
Feature Request: Multiloading scripts

### ISSUE THREAD

**Issue by nfmuss** [FredTungsten/ScriptPlayer#252]
Hello,

I would like to request the ability to load multiple scripts for one video to allow the user to switch to different scripts during playback.
This could allow for the user to switch from a low intensity script to a higher intensity script without interrupting the video playback.

thank you

**Comment by nfmuss** [comment]
An alternative method would be to add a [Go Forward/backward 10 Minutes] function. 
I could render the same clip twice in one video file, and use the +-10min function to skip to the same repeated section of the video but with a higher intensity script. 

For example, for a 7:30 minute video, I would turn it into a 17:30 minutes video. 
First playback(00:01~07:30) would have a low intensity script.
Blank section(7:31~10:00)
Then the same video(10:01~17:30) would have a high intensity script. 

At say around the 4:30 minute mark, I wanted to change to a higher intensity script I would just use the +10minute function to skip to 14:30, which would be the same part of the video except with higher intensity script, and I could switch back too with a -10min function. 


I apologize if I’m not making sense :$ 
To summarize it would be great to be able to load two scripts to switch back and forth from without interrupting the video playback, otherwise a function to skip forward 10 minutes, instead of only 5 seconds, would be amazing.

**Comment by FredTungsten** [comment]
Might happen, but likely not anytime soon

**Comment by rbwabd** [comment]
Hi FredTungsten,

i actually implemented something that would allow this to happen already. Sent you a DM on your milanova account regarding how to make it public since github doesn't allow messages, could you please check thx.

**Comment by rbwabd** [comment]
nfmuss, i've hacked together some scripts that can handle what you are looking for. see here: https://discuss.eroscripts.com/t/autoit-scripts-for-adding-features-to-scriptplayer-for-pmv-funscripts/97771

**Comment by nfmuss** [comment]
Thank you rbwabd, this looks really comprehensive, I appreciate it a lot!
